app-modules/form/src/Filament/Resources/FormResource/Pages/CreateForm.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Assist\Form\Filament\Resources\FormResource\Pages;
+
+use Filament\Forms\Form;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Resources\Pages\CreateRecord;
+use Assist\Form\Filament\Resources\FormResource;
+
+class CreateForm extends CreateRecord
+{
+    protected static string $resource = FormResource::class;
+
+    public function form(Form $form): Form
+    {
+        return parent::form($form)
+            ->schema([
+                TextInput::make('name')
+                    ->required()
+                    ->string()
+                    ->maxLength(255)
+                    ->autocomplete(false)
+                    ->columnSpanFull(),
+                Textarea::make('description')
+                    ->string()
+                    ->columnSpanFull(),
```

Comment:
Why do we not have the form field builder on create?

Reply:
I thought it was confusing to do it all at once on create, but I've added it now.

Also, we're going to need a way to check for unique on the fields. I tried using both the unique rule and the rules call, but I can't figure it out because it needs to take the key and form_id into account, both on edit and on creation.